frontend/src/hooks/useExceptionHandler.jsx

```diff
@@ -26,6 +26,13 @@ const useExceptionHandler = () => {
           // Handle validation errors
           if (setBackendErrors) {
             setBackendErrors(err?.response?.data);
+          } else {
+            return {
+              title: title,
+              type: "error",
+              content: errors?.[0]?.detail ? errors[0].detail : errMessage,
```

Comment:
@johnyrahul why is this required? AFAIK we use a library to format the serializer errors. Can you provide a screenshot of what error we're trying to fix here?

Reply:
@
![image](https://github.com/user-attachments/assets/44182417-2a3b-49ec-abf2-3ae99cf4cce4)

I have discussed this with @tahierhussain . Yes we can improve this to handle all error messages in the list.